### ISSUE TITLE
add an option to recover deleted projects as from the view of a creator sometime people get angry and delete thier precious work

### ISSUE THREAD

**Issue by beamty** [thunkable/thunkable-issues#1166]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Which platform are you asking for?**
Thunkable Classic (app.thunkable.com) or Thunkable ✕ (x.thunkable.com)

**Additional context**

**Comment by beamty** [comment]
and the solution for this is to add a cloud source which can be used from firebase servers storing hich helps people to regain their time precious project

**Comment by tatiang** [comment]
Yes, this has been requested before. Either a restore project feature or -- I think even better -- version history support.

https://github.com/thunkable/thunkable-issues/issues/1062
https://github.com/thunkable/thunkable-issues/issues/674

The current workaround is to save a copy (or several copies) of each project as you make changes. This is my approach but it's hard to manage and it very quickly uses up all of your available project quota.

**Comment by weihuali0509** [comment]
@tatiang thanks for the summary.
